app/models/content_item.rb

```diff
@@ -20,6 +22,8 @@ def initialize(content_store_response, override_content_store_hash: nil)
     @base_path = content_store_hash["base_path"]
     @locale = content_store_hash["locale"]
     @public_updated_at = content_store_hash["public_updated_at"]
+    @first_published_at = content_store_hash["first_published_at"]
+    @first_public_at = content_store_hash.dig("details", "first_public_at") || first_published_at
```

Comment:
Do we know how many different schemas have a `first_public_at`? (is it only case studies?)